### ISSUE TITLE
Move all jinja2 logic to YAML

### ISSUE THREAD

**Issue by nvkelso** [tilezen/vector-datasource#1356]
1. There are a few pieces that are better handled in layer YAML config than the jinja2.

https://github.com/tilezen/vector-datasource/blob/master/queries/planet_osm_point.jinja2#L26-L32

```
  CASE WHEN mz_places_min_zoom IS NOT NULL
    THEN jsonb_build_object(
      'country_capital', CASE WHEN tags->'capital' = 'yes' THEN TRUE END,
      'region_capital', CASE WHEN tags->'state_capital' = 'yes' THEN TRUE END,
      'min_zoom', mz_places_min_zoom
    )
  END AS __places_properties__,
```

2. There are many cases when we're setting up query ranges per table in the JINJA (especially Natural Earth) that we can move to the queries.yaml:

https://github.com/tilezen/vector-datasource/blob/master/queries.yaml#L12-L28

**Comment by zerebubuth** [comment]
Generalising this to cover all the cases where there are SQL expressions like the example above. We should move across all the expressions where we can comfortably accommodate it in the Jinja2, e.g: all the stuff that doesn't involve joining to other tables, etc...

**Comment by nvkelso** [comment]
Mostly resolved with #1362.

**Comment by zerebubuth** [comment]
There's still a lot of places where we have logic in SQL that, ideally, we'd have in YAML instead. However, I'm not sure that this would be on any kind of near-term roadmap, so I don't think it should be re-opened.
